README.md

```diff
@@ -808,16 +808,32 @@ With using a company management key you can get a list of all the projects in th
 const projects = await descopeClient.management.project.listProjects();
 ```
 
-You can manage your project's settings and configurations by exporting your
-project's environment. You can also import previously exported data into
-the same project or a different one.
+You can manage your project's settings and configurations by exporting a snapshot:
 
 ```typescript
 // Exports the current state of the project
-const files = await descopeClient.management.project.export();
+const exportRes = await descopeClient.management.project.exportSnapshot();
+```
+
+You can also import previously exported snapshots into the same project or a different one:
+
+```typescript
+const validateReq: ValidateSnapshotRequest = {
```

Comment:
I don't think that this type is exported, you exported it from types file, but not from the index file
this typing things is a big thing, we can probably discuss this for days about how to achieve that, but we are not there yet :) for now lets just remove the type from doc

Reply:
I've updated the README. Do we usually export misc types like these? Or is it enough for normal use that they are defined and then available during coding via autocomplete?

Reply:
I'm not sure how / where it is used , usually such data is defined on the return value , and we don't export most of it 

Also note that if you don't export from index.ts then it's not exported 

I would remove it for now